db/models/user.js

```diff
@@ -2,27 +2,48 @@
 
 // bcrypt docs: https://www.npmjs.com/package/bcrypt
 const bcrypt = require('bcryptjs')
-    , {STRING, VIRTUAL} = require('sequelize')
+    , {BOOLEAN, STRING, VIRTUAL} = require('sequelize')
 
 module.exports = db => db.define('users', {
-  name: STRING,
+  userName: STRING,
   email: {
     type: STRING,
     validate: {
       isEmail: true,
-      notEmpty: true,
     }
   },
+  firstName: STRING,
+  lastName: STRING,
+  shippingAddress: STRING ,
+  creditInfo: {
```

Comment:
Similar question to above -- is credit card info really stored as just an integer? I would think you'd need separate fields for the number, the CVC, and the billing address. On that note, I think we'll also want a separate field for billing address -- right now you have it generating from the shipping address via a getterMethod, but users may have separate billing and shipping addresses.